tests/src/limits/ThrottleTests.scala

```diff
@@ -174,23 +175,25 @@ class ThrottleTests
                 (action, _) => action.create(name, timeoutAction)
             }
 
-            val slowInvokes = maximumConcurrentInvokes * 0.4
-            val fastInvokes = maximumConcurrentInvokes * 0.6 + 10
+            val slowInvokes = maximumConcurrentInvokes * 0.6
+            val fastInvokes = maximumConcurrentInvokes * 0.4 + 1
 
             // Keep queue from draining with these
             val slowResults = untilThrottled(slowInvokes.toInt) { () =>
-                wsk.action.invoke(name, Map("payload" -> 10.seconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
+                wsk.action.invoke(name, Map("payload" -> 15.seconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
             // Create queue length quickly, drain fast
             val fastResults = untilThrottled(fastInvokes.toInt) { () =>
                 wsk.action.invoke(name, Map("payload" -> 10.milliseconds.toMillis.toJson), expectedExitCode = DONTCARE_EXIT)
             }
 
-            // Sleep 3 seconds to let the background thread get the newest values
+            // Sleep 5 seconds to let the background thread get the newest values (refreshes every 2 seconds)
             Thread.sleep(3.seconds.toMillis)
 
-            val endResults = untilThrottled(10) { () =>
+            // Start only 119 activations to avoid being blocked by the invokesPerMinute limit
```

Comment:
`// start 1 invoke less than the maximum per minute to avoid getting rate throttled`